README.md

```diff
@@ -203,6 +203,19 @@ Follow the instructions in the error message to resolve the mismatch.
 #### developer
 If you are building `probe-run` from source, you can disable the version check by setting the `PROBE_RUN_IGNORE_VERSION` environment variable to `true` or `1` at compile time.
 
+
+### developer: running your locally modified `probe-run`
+
```

Comment:
Maybe we don't need the headline and attach it into the section just before?

```suggestion
```

Reply:
they serve different purposes though– I just added that particular environment variable because it was part of one of my most recent local probe-run runs 😅

Reply:
Okay :)